packages/coinstac-ui/app/render/components/consortia/consortia-list.jsx

```diff
@@ -150,10 +150,7 @@ class ConsortiaList extends Component {
           classNames(classes.value, consortium.activePipelineId ? classes.green : classes.red)
         }
         >
-          {consortium.activePipelineId
-            ? pipeline.name
-            : 'None'
-          }
+          { pipeline ? pipeline.name : 'None' }
```

Comment:
?

Reply:
I changed this because the `pipelines` prop that comes from the dashboard and is basically a GraphQL subscription to the pipelines, might be undefined or an empty array in the first few renders of the app.

This was not the case before because we were using the Apollo HOC to wrap the dashboard component and that only rendered the component when the first query for all subs had resolved. That is not the case anymore, since I'm now using hooks (which are the recommended way now).